generator/dotnet/template/Readme.md

```diff
@@ -0,0 +1,120 @@
+# Affinidi Login Reference App 
+
+This project is a .NET 8.0 Razor web app, utilizing the power of Razor Pages for building dynamic web pages.
+
+## Getting Started
+
+These instructions will help you to make this project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+Make sure you have the following tools installed on your machine:
+
+- [.NET SDK 8.0](https://dotnet.microsoft.com/download)
+- [Visual Studio code](https://code.visualstudio.com/) (or any preferred code editor)
+
+    #### PreRequisite check 
+
+    Run this command to get the Dotnet version 
+    ```sh 
+    dotnet --version 
+    ```
+
+    You should get Output as, please reinstal if dotet version is not 8.0.xxx
```

Comment:
please change to reinstall and dotnet. You can setup a spelling check to find others as well

Reply:
changed